ci/playbooks/tcib/tcib.yml

```diff
@@ -0,0 +1,14 @@
+---
+- name: Bootstrap step
+  ansible.builtin.import_playbook: "{{ ansible_user_dir }}/src/github.com/openstack-k8s-operators/ci-framework/ci_framework/playbooks/01-bootstrap.yml"
+
+- hosts: "{{ cifmw_target_host | default('localhost') }}"
+  gather_facts: true
+  tasks:
+    - name: Deploy local registry
+      ansible.builtin.include_role:
+        name: registry_deploy
```

Comment:
Add the following task here to set [tag](https://logserver.rdoproject.org/85/285/b066a5fd9c989d19c4de05b0ecee5370e8a158fa/github-check/cifmw-base-tcib/0d190cc/ci-framework-data/logs/build.log)
```
- name: Read hash from delorean.repo.md5 file
      ansible.builtin.slurp:
        path: "{{ cifmw_basedir }}/artifacts/repositories/delorean.repo.md5"
      register: file_content

    - name: Set fact for cifmw_build_containers_image_tag: var
      ansible.builtin.set_fact:
        cifmw_build_containers_image_tag: "{{ file_content['content'] | b64decode | string | trim }}"
```